py/emitnative.c

```diff
@@ -288,6 +288,14 @@ struct _emit_t {
     ASM_T *as;
 };
 
+#if N_RV32
+#define REG_ZERO ASM_RV32_REG_ZERO
+#define CLEAR_REGISTER(as, reg)
+#else
+#define REG_ZERO REG_TEMP0
+#define CLEAR_REGISTER(as, reg) ASM_XOR_REG_REG(as, reg, reg)
+#endif
```

Comment:
I feel like this should go in `py/asm*.h`, alongside the other `REG_xxx` definitions.  And only rv32 needs to define it, then here it could be:
```c
#ifndef REG_ZERO
#define REG_ZERO REG_TEMP0
...
#endif
```
Similar to how `REG_PARENT_RET` is defined in this file.

Reply:
Fair point.  This should be addressed in the current revision of the PR.